.circleci/config.yml

```diff
@@ -39,6 +39,7 @@ jobs:
             - node_modules
             - build
             - build-info.json
+            - static/stylesheets
```

Comment:
why do we need this line - I thought `webpack` would put it into `build` directory anyway?